docs/README.md

```diff
@@ -0,0 +1,15 @@
+# Welcome
+**Here are SkyWalking Infra E2E official documentations. Welcome to use it.**
+
+SkyWalking Infra E2E is an End-to-End Testing framework that aims to help developers to set up, debug, and verify E2E tests with ease.
+It’s built based on the lessons learned from tens of hundreds of test cases in the SkyWalking main repo.
+
+From here you can learn all about **SkyWalking Infra E2E's** architecture, how to set up E2E testing.
+
+- [Concepts and Designs](en/concepts-and-designs/README.md). The most important core ideas about SkyWalking Infra E2E. You can learn from here if you want to understand what is going on under our cool features.
```

Comment:
Maybe we should link to https://skywalking.apache.org/blog/e2e-design/?

Reply:
Sure, but it has some differences, such as we don't have the code generator. Should we also delete project goals if we use the https://skywalking.apache.org/blog/e2e-design/ link?